LayoutTests/svg/parser/whitespace-angle-1-expected.txt

```diff
@@ -0,0 +1,530 @@
+PASS Test <angle> valid value: 1e-10\f
+PASS Test <angle> valid value: 1e-10deg\f
+PASS Test <angle> valid value: 1e-10rad\f
+PASS Test <angle> valid value: 1e-10grad\f
+PASS Test <angle> trailing garbage, value: -47a
+PASS Test <angle> trailing garbage, value: .1a
+PASS Test <angle> trailing garbage, value: 0.35a
+PASS Test <angle> trailing garbage, value: 1e-10a
+PASS Test <angle> trailing garbage, value: -47e
+PASS Test <angle> trailing garbage, value: .1e
+PASS Test <angle> trailing garbage, value: 0.35e
+PASS Test <angle> trailing garbage, value: 1e-10e
+PASS Test <angle> trailing garbage, value: -47foo
+PASS Test <angle> trailing garbage, value: .1foo
+PASS Test <angle> trailing garbage, value: 0.35foo
+PASS Test <angle> trailing garbage, value: 1e-10foo
+PASS Test <angle> trailing garbage, value: -47)90
+PASS Test <angle> trailing garbage, value: .1)90
+PASS Test <angle> trailing garbage, value: 0.35)90
+PASS Test <angle> trailing garbage, value: 1e-10)90
+PASS Test <angle> valid value:  -47
+PASS Test <angle> valid value:  -47deg
+PASS Test <angle> valid value:  -47rad
+PASS Test <angle> valid value:  -47grad
+PASS Test <angle> valid value:  .1
+PASS Test <angle> valid value:  .1deg
+PASS Test <angle> valid value:  .1rad
+PASS Test <angle> valid value:  .1grad
+PASS Test <angle> valid value:  0.35
+PASS Test <angle> valid value:  0.35deg
+PASS Test <angle> valid value:  0.35rad
+PASS Test <angle> valid value:  0.35grad
+PASS Test <angle> valid value:  1e-10
+PASS Test <angle> valid value:  1e-10deg
+PASS Test <angle> valid value:  1e-10rad
+PASS Test <angle> valid value:  1e-10grad
+FAIL Test <angle> valid value:  -47  assert_approx_equals: expected -47 +/- 5.960464477539063e-8 but got 0
+FAIL Test <angle> valid value:  -47deg  assert_approx_equals: expected -47 +/- 5.960464477539063e-8 but got 0
+FAIL Test <angle> valid value:  -47rad  assert_approx_equals: expected -47 +/- 5.960464477539063e-8 but got 0
+FAIL Test <angle> valid value:  -47grad  assert_approx_equals: expected -47 +/- 5.960464477539063e-8 but got 0
+FAIL Test <angle> valid value:  .1  assert_approx_equals: expected 0.1 +/- 5.960464477539063e-8 but got 0
+FAIL Test <angle> valid value:  .1deg  assert_approx_equals: expected 0.1 +/- 5.960464477539063e-8 but got 0
+FAIL Test <angle> valid value:  .1rad  assert_approx_equals: expected 0.1 +/- 5.960464477539063e-8 but got 0
+FAIL Test <angle> valid value:  .1grad  assert_approx_equals: expected 0.1 +/- 5.960464477539063e-8 but got 0
+FAIL Test <angle> valid value:  0.35  assert_approx_equals: expected 0.35 +/- 5.960464477539063e-8 but got 0
+FAIL Test <angle> valid value:  0.35deg  assert_approx_equals: expected 0.35 +/- 5.960464477539063e-8 but got 0
+FAIL Test <angle> valid value:  0.35rad  assert_approx_equals: expected 0.35 +/- 5.960464477539063e-8 but got 0
+FAIL Test <angle> valid value:  0.35grad  assert_approx_equals: expected 0.35 +/- 5.960464477539063e-8 but got 0
```

Comment:
Do you know why the code fails for these?

specifically given that similar values pass.